7. Regular Expressions.js

```diff
@@ -16,4 +16,23 @@ let now = new Date();
 let year = now.getFullYear();
 let month = now.getMonth();
 let day = now.getDate();
-console.log(`${day}\n${month}\n${year}`);
+
+function getCurrentMonth(month) {
+  let months = [
+    "jan",
+    "feb",
+    "march",
+    "apr",
+    "may",
+    "jun",
+    "july",
+    "aug",
+    "sept",
+    "oct",
+    "nov",
+    "dec",
+  ];
+  return months[month];
+}
+
+console.log(`${day}\n${getCurrentMonth(month)}\n${year}`);
```

Comment:
you don't need to make a specific function `getCurrentMonth()`.

In line 17 enough to make `let month = now.getMonth() + 1;`
And here just to print `${day}\n${month}\n${year}`.